compiler/rustc_middle/src/hir/map/collector.rs

```diff
@@ -358,6 +358,9 @@ impl<'a, 'hir> Visitor<'hir> for NodeCollector<'a, 'hir> {
     }
 
     fn visit_generic_param(&mut self, param: &'hir GenericParam<'hir>) {
+        if let hir::GenericParamKind::Const { default: Some(ref default), .. } = param.kind {
+          self.visit_anon_const(default);
+        }
```

Comment:
we have to update `intravisit::walk_generic_param` here

Reply:
ah hm I think I already updated that file with the change tho, it would be visiting it twice now which would cause the bug